frontend/src/views/components/Journey/Embed/Embed.component.jsx

```diff
@@ -70,7 +75,9 @@ const Embed = (props) => {
 
   const provideAbsoluteOrRelativeUrl = (url) => {
     if (url.startsWith('http')) {
-      return url.replace('resilienceatlas.org/', `resilienceatlas.org/${locale}/`);
+      return url
+        .replace(/(:\/\/)(?!www.)([^\/]+)/, `://www.$2`) // Temporary solution until redirects are fixed. Use regex to add www. to url
```

Comment:
This gives security warnings on Firefox:

![The embed displays a “Potential Security Issue” warning](https://user-images.githubusercontent.com/6073968/236458428-f696034f-067d-44da-bc27-fa012c28d9c9.png)


Instead, maybe you could just replace `'//resilienceatlas.org'` by `'//www.resilienceatlas.org'`? The “www” issue only affects the main domain.

Reply:
Actually is affecting all the subdomains too: https://madagascar.resilienceatlas.org/
Does this work for you?

Reply:
Yes, it loads perfectly on Firefox and Chrome. It doesn't for you? Did you forget to reset your `/etc/hosts` by any chance?